packages/gatsby/src/redux/actions/public.ts

```diff
@@ -41,17 +75,17 @@ const {
   getInProcessJobPromise,
 } = require(`../../utils/jobs-manager`)
 
-const actions = {}
+export const actions: Record<string, any> = {}
 const isWindows = platform() === `win32`
 
-const ensureWindowsDriveIsUppercase = filePath => {
+const ensureWindowsDriveIsUppercase = (filePath): string => {
   const segments = filePath.split(`:`).filter(s => s !== ``)
   return segments.length > 0
     ? segments.shift().toUpperCase() + `:` + segments.join(`:`)
     : filePath
 }
 
-const findChildren = initialChildren => {
+const findChildren = (initialChildren): any[] => {
```

Comment:
missing typeDef on initialChildren, can we improve the `any` ?

Reply:
Done in a1b5cbd